isi_data_insights_config.py

```diff
@@ -74,9 +75,11 @@ def _process_config_file_clusters(clusters):
         if len(at_split) == 2:
             user_pass_split = at_split[0].split(":", 1)
             if len(user_pass_split) != 2:
-                print >> sys.stderr, "Config file contains invalid cluster " "config: %s in %s (expected <username>:<password> " "prefix)." % (
-                    cluster_config,
-                    clusters,
+                print(
+                    "Config file contains invalid cluster "
+                    "config: %s in %s (expected <username>:<password> "
+                    "prefix)." % (cluster_config, clusters),
```

Comment:
Alternatively
``` python
print(
    "Config file contains invalid cluster config:",
    cluster_config, "in", clusters,
    "(expected <username>:<password> prefix).",
    file=sys.stderr,
)
```

Reply:
Yes, these all need tidying up. I simply ran "black" over the code base but I want to do another pass tidying up the print statements (and really move over to using a logger anyway).